cleverhans/attacks.py

```diff
@@ -226,6 +236,9 @@ class FastGradientMethod(Attack):
     def __init__(self, model, back='tf', sess=None):
         """
         Create a FastGradientMethod instance.
```

Comment:
It's more straightforward to just say `cleverhans.model.Model`.

Rather than documenting how the callable should behave, maybe we should just leave that out. If people want to duck-type the `Model` class, its call method returns probabilities, so they should do the right thing when reading its doc.

In places were we require probabilities, we can usually check that the output comes from some specific op, usually Softmax. Explicit runtime checks are a lot more reliable than documentation.